c_emulator/version.cpp.in

```diff
@@ -0,0 +1,5 @@
+#include "version.h"
+
+const char *get_version() {
+  return "@sail_riscv_version@";
+}
```

Comment:
I don't feel strongly about any of this but wouldn't it be simpler to just have a single version.h.in header containing

```
#define SAIL_RISCV_VERSION "@sail_riscv_version@" and then update the printf to use that directly?

Reply:
It would be simpler.  It's just overoptimizing trying to prevent `riscv_sim.cpp` being compiled when `version.h` changed.  I can make it a single header if you prefer.

Reply:
Ok, simplified to a single header.